scripts/publish-site.sh

```diff
@@ -11,6 +11,11 @@ yarn
 echo "=== Copying built Gatsby to website."
 gatsby-dev --scan-once --quiet
 
+echo "=== find graphql packages"
+
+find node_modules -name graphql
+echo "=== END find graphql packages"
```

Comment:
Did that already... :-\

Can't reproduce locally — it's giving me a weird graphql error where it says the "schema" isn't the correct type.

Breaking out Docker...